astropy/utils/data.py

```diff
@@ -910,54 +910,67 @@ def _find_hash_fn(hexdigest, pkgname='astropy'):
     return None
 
 
-def get_free_space_in_dir(path):
+def get_free_space_in_dir(path, unit=False):
```

Comment:
@mhvk , I think I applied all your suggestions, except setting default to `False` instead of `None`. I think `False` is more consistent (staying in the realm of `bool`) when we start to support `True`.